### PR TITLE
[feature](invert index) match_phrase_prefix feature added

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

select count() from test_index_match_phrase_prefix where request match_phrase_prefix 'xxx';

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/vec/functions/match.h [hunk 1]
```diff
@@ -128,6 +128,23 @@ class FunctionMatchPhrase : public FunctionMatchBase {
...
+    static FunctionPtr create() { return std::make_shared<FunctionMatchPhrasePrefix>(); }
+
+    String get_name() const override { return name; }
+
+    Status execute_match(const std::string& column_name, const std::string& match_query_str,
+                         size_t input_rows_count, const ColumnString* string_col,
+                         InvertedIndexCtx* inverted_index_ctx,
+                         const ColumnArray::Offsets64* array_offsets,
+                         ColumnUInt8::Container& result) const override {
+        return Status::Error<ErrorCode::INVERTED_INDEX_NOT_SUPPORTED>(
```
Comment (airborne12) [inline be/src/vec/functions/match.h:143]: may be we need to support

**Review by airborne12 — APPROVED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Review by airborne12 — APPROVED** [review]

**Pull request merged**